crowbar_framework/Gemfile

```diff
@@ -58,7 +58,7 @@ gem "rack-mini-profiler", "~> 0.9",
 
 group :development, :test do
   gem "brakeman", "~> 2.6"
-  gem "rspec-rails", "~> 2.14"
+  gem "rspec-rails", "~> 3.1"
```

Comment:
Why do you care about the versions in the Gemfile? You are not using it anyway. And if we pin the versions too hard we get problems for updates as we always have to ship a new crowbar version instead of just updating the gem rpm. I have not done this version scheme just out of fun...